README.md

```diff
@@ -1,5 +1,27 @@
 # FHIR Mock Data
 
+## Timewarp
+
+Used to keep demo data in sync, the timewarp utility will advance all date and
+datetime attributes found in the named directory of FHIR Resource files, a 
+given number of days and then PUT the changed files to the named HAPI store.
+
+Example use:
+
+Copy `fhir-walk.py` into the `timewarp` container and export current resources:
+```
+docker cp ~/fhir-walk/fhir-walk.py `docker ps|grep timewarp|awk '{print $1}'`:.
+docker-compose exec timewarp bash
```

Comment:
Can you rewrite these `docker compose exec` as `docker compose run`? Happy to help, but timewarp seems like a one-off job vs a persistent service that needs to keep running

Reply:
I believe that would require an accessible volume.  As is, don't we need the same container (which i believe exec gives vs. run?) to first generate the fhir-walked ndjson files, and then the second to process?

Or would it be a better idea to create a separate container to write the files to a shared volume as opposed to copying in an executable?

Reply:
given that we don't need this utility w/o on older versions of HAPI where bulk $export isn't available, i opted to instead bring [your bulk export](https://github.com/uwcirg/fhir-eval-environments/blob/main/utils/fhir-server-export.py) script into the fold.

thanks to @ivan-c 's help, we have now completely decoupled this solution from its initial target.  See the example in the README which pulls and runs a container built from this repo on the given parameters.